src/Raven.Server/Documents/ChangesClientConnection.cs

```diff
@@ -506,15 +634,71 @@ public DynamicJsonValue GetDebugInfo()
                 ["SubProtocol"] = _webSocket.SubProtocol,
                 ["Age"] = Age,
                 ["WatchAllDocuments"] = _watchAllDocuments > 0,
-                ["WatchAllIndexes"] = false,
-                /*["WatchConfig"] = _watchConfig > 0,
-                ["WatchConflicts"] = _watchConflicts > 0,
-                ["WatchSync"] = _watchSync > 0,*/
+                ["WatchAllIndexes"] = _watchAllIndexes > 0,
+                ["WatchAllCounters"] = _watchAllCounters > 0,
+                ["WatchAllOperations"] = _watchAllOperations > 0,
                 ["WatchDocumentPrefixes"] = _matchingDocumentPrefixes.ToArray(),
                 ["WatchDocumentsInCollection"] = _matchingDocumentsInCollection.ToArray(),
                 ["WatchIndexes"] = _matchingIndexes.ToArray(),
-                ["WatchDocuments"] = _matchingDocuments.ToArray()
+                ["WatchDocuments"] = _matchingDocuments.ToArray(),
+                ["WatchCounters"] = _matchingCounters.ToArray(),
+                ["WatchCounterOfDocument"] = _matchingDocumentCounter.Select(x => x.ToJson()).ToArray(),
+                ["WatchCountersOfDocument"] = _matchingDocumentCounters.ToArray()
             };
         }
+
+        private static DocumentIdAndCounterNamePair GetDocumentCounterParameters(string parameter)
```

Comment:
We not send this properly, as two arguments?

Reply:
Fixed